src/main/java/org/kafkahq/service/ConsumerGroupService.java

```diff
@@ -49,16 +52,31 @@ public ConsumerGroupService(KafkaModule kafkaModule, ConsumerGroupMapper consume
         this.recordRepository = recordRepository;
     }
 
-    public ConsumerGroupListDTO getConsumerGroup(String clusterId, String view,Optional<String>  search, Optional<Integer> pageNumber)
+    public ConsumerGroupListDTO getConsumerGroup(String clusterId, String view, Optional<String> search, Optional<Integer> pageNumber)
             throws ExecutionException, InterruptedException {
         Pagination pagination = new Pagination(pageSize, pageNumber.orElse(1));
 
         PagedList<ConsumerGroup> list = this.consumerGroupRepository.list(clusterId, pagination, search);
-        ArrayList<ConsumerGroupDTO>consumerGroupList=new ArrayList<>();
+        ArrayList<ConsumerGroupDTO> consumerGroupList = new ArrayList<>();
         list.stream().map(consumerGroup -> consumerGroupList.add(consumerGroupMapper.fromConsumerGroupToConsumerGroupDTO(consumerGroup))).collect(Collectors.toList());
 
         return new ConsumerGroupListDTO(consumerGroupList, list.total());
     }
 
+//Precorrer a lista de offsets e aplicar o mapper e depois retorna a lista
+    public List<ConsumerGroupOffsetDTO> getConsumerGroupOffsets(String clusterId, String groupId) throws ExecutionException, InterruptedException {
+
+        ConsumerGroup group = this.consumerGroupRepository.findByName(clusterId, groupId);
+        List<TopicPartition.ConsumerGroupOffset> offsets= group.getOffsets();
+        List<ConsumerGroupOffsetDTO> offsetsDTO= new ArrayList<>();
+        for(int i=0; i<offsets.size();i++){
+           ConsumerGroupOffsetDTO offsetDTO = consumerGroupMapper.fromConsumerGroupToConsumerGroupOffsetDTO(offsets.get(i));
+            offsetsDTO.add(offsetDTO);
+        }
```

Comment:
Podes fazer este for com um .stream().map() para ficar mais legível, mas é só uma sugestão. Podes ver, como exemplo, este método:
![image](https://user-images.githubusercontent.com/28408419/76630633-45b0a580-6538-11ea-9c8f-aeae15325ab5.png)